### ISSUE TITLE
Minimal database for electron app

### ISSUE THREAD

**Issue by tjayrush** [Great-Hill-Corporation/trueblocks-tokenomics#123]
Currently, we store all the transactions with full detail including all the transaction traces in a mySQL database so that we can write what I call a web 2.0 interface in front of it, but this does not work well with the idea of "minimal impact on the target machine."

I have a test case that shows how fast it is to pull transactions from a running Ethereum node (it's way faster to pull from a locally running node than from a remote Infura node, and way faster still to use QuickBlocks cache than a locally running node), but in both cases it's reasonably fast. I would like to explore what it would look like if the electron desktop app ran directly against the node given a super small list of block.transactionsIndex numbers.

In other words, minimal impact on the machine.

The only question then becomes how does one deliver the blockNumber.transactionIndex pair. Solution: a fully customized (i.e. much less general) version of TrueBit called TrueBlocks that delivers blockNumber.transaction pairs for given addresses and rewards people for "garbage collection"
